core/src/main/java/org/jdbi/v3/core/mapper/reflect/JdbiConstructors.java

```diff
@@ -27,6 +28,31 @@ private JdbiConstructors() {
         throw new UnsupportedOperationException("utility class");
     }
 
+    static <T> InstanceFactory<T> findFactoryFor(Class<T> type) {
```

Comment:
Let's make this method public, document it, and deprecate `findConstructorFor(Class)`